job-dsls/jobs/scrclr_scan_seed_job.groovy

```diff
@@ -0,0 +1,26 @@
+job('srcclr_scan_seed_job') {
+    description('Scan job, which generates scanning jobs for upstream projects')
+    parameters {
+        stringParam('REPO_FILE_URL','https://raw.githubusercontent.com/akoufoudakis/droolsjbpm-build-bootstrap/BXMSPROD-533/script/repository-list.txt','URL of the rpository-list.txt file')
+        stringParam('KIE_JENKINS_SCRIPTS_REPO', 'https://github.com/akoufoudakis/kie-jenkins-scripts.git', '')
+        stringParam('KIE_JENKINS_SCRIPTS_BRANCH', 'master', '')
+        stringParam('JOB_PATH', 'custom/akoufoud/source-clear/srcclr', '')
```

Comment:
this is pointing to your custom folder, I would remove the parameter and use an environment variable for this (in order to be able to move it anywhere we want with no need to changes this)

Reply:
You're right. Defaults have to be kiegroup repos.

Reply:
I don't think we should hide this to jenkins enviroment var. We can always regenerate it with changed value parameter, but the default value should be not custom/... ;) 
Default should be based on branch, so PROD/<branch>/...